mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -188,65 +276,121 @@ export class CompareRunView extends Component {
                       ? Utils.formatTimestamp(run.getStartTime())
                       : '(unknown)';
                     return (
-                      <td className='meta-info' key={run.run_uuid}>
-                        {startTime}
+                      <td className='data-value' key={run.run_uuid} style={colWidthStyle}>
+                        <Tooltip
+                          title={startTime}
+                          color='blue'
+                          overlayStyle={{ maxWidth: '400px' }}
+                        >
+                          {startTime}
+                        </Tooltip>
                       </td>
                     );
                   })}
                 </tr>
+              </tbody>
+              <tbody className='table-block' style={tableBlockWidthStyle}>
                 <tr>
                   <th
                     scope='rowgroup'
-                    className='inter-title'
+                    className='inter-title sticky_header'
                     colSpan={this.props.runInfos.length + 1}
+                    style={tableBlockWidthStyle}
                   >
-                    <h2>
-                      <FormattedMessage
-                        defaultMessage='Parameters'
-                        // eslint-disable-next-line max-len
-                        description='Row group title for parameters of runs on the experiment compare runs page'
+                    <CollapsibleSection
+                      title={
+                        <FormattedMessage
+                          defaultMessage='Parameters'
+                          // eslint-disable-next-line max-len
+                          description='Row group title for parameters of runs on the experiment compare runs page'
+                        />
+                      }
+                      onChange={(activeKeys) =>
+                        this.setState({ collapseParamBlock: activeKeys.length === 0 })
+                      }
+                    >
+                      <Switch
+                        checkedChildren='Show diff only'
+                        unCheckedChildren='Show diff only'
+                        onChange={(isShowDiffOnly, e) =>
```

Comment:
```suggestion
                        onChange={(checked, e) =>
```

I prefer using `checked` here since it describes what this argument represents.